packages/shared/src/internalApi/api.ts

```diff
@@ -1,23 +1,23 @@
 import fetch, { RequestInit } from 'node-fetch';
-import { getInternalApiOrigin } from '../utils/env';
+import { getGradeJsApiKey, getInternalApiRootUrl } from '../utils/env';
 
 export type DetectedPackage = {
   name: string;
-  possibleVersions: string[];
+  versionSet: string[];
   versionRange: string;
```

Comment:
It seems like we don't need `versionRange` anymore

Reply:
Keeping it for backwards-compatibility for now, until we rework the scan data structure itself